### PR TITLE
[Japanese] Fix Grammer Problem 

### PR DESCRIPTION
**Pull request by peyang-Celeron**
# [Japanese] Long vowels unification

## Description
+ This PR fixes a problem with long vowels in #220.
+ Unified "er" at the word end with "ー".
+ The notation of "ye - r" in words has been unified to "ー".
+ This PR is a compilation of #222 and #223.

## Issues
+ [#220 \[Japanese\] Fixing incomplete translations and translations that don't match the guide.](https://github.com/mty22/krunker.io-translations/issues/220)

### CODE REVIEW

**Review by Hiro527 — DISMISSED** [review]
修正ありがとうございます。
確認したところ特に問題は無いと思います。

**Comment by peyang-Celeron** [comment]
了解いたしました。  
迅速なレビューありがとうございます。

**Comment by Hiro527** [comment]
@mty22 
I've already checked the changes. It's fixed about Japanese grammar problems.
There's no problem to merge. I'm waiting for your review but no rush to do this:)

@peyang-Celeron 
プロジェクトの管理者兼リポジトリオーナーのmattさんのapproveなどがないとマージできないのでしばらくお待ち下さい。
多分このまま通ると思います。

**Comment by mty22** [comment]
@Hiro527 Alright, no problems. I consider you owner of `jp.js` file, so if you think it's good to go then I'll verify the changes and merge.

@peyang-Celeron Thanks for getting involved! :) can you please note down your Krunker IGN and Discord ID so I can queue up KR to be sent, and adjust roles in KrunkerBunker? Also, feel free to include yourself in README.md and CODEOWNERS (you can update and push to this PR if you wish).

**Comment by peyang-Celeron** [comment]
> @peyang-Celeron Thanks for getting involved! :) can you please note down your Krunker IGN and Discord ID so I can queue up KR to be sent, and adjust roles in KrunkerBunker? Also, feel free to include yourself in README.md and CODEOWNERS (you can update and push to this PR if you wish).

Thanks for your response. My Discord id is `Peynaf#0001` and Krunker IGN is `peyang`.
I'm going to edit the list of contributors in README so I'm sorry if the review approve status is changed.

File: README.md
Comment (Hiro527) [inline README.md:74]:
```md
[peyang-Celeron](https://krunker.io/social.html?p=peyang) (Peyang#0001)
```
に修正お願いします

Comment (peyang-Celeron) [reply README.md]: 修正いたしました。

**Comment by mty22** [comment]
@Hiro527 all good to merge?

**Comment by Hiro527** [comment]
@mty22 yes, everything okay:)

**Comment by mty22** [comment]
kr queued to be sent.

**Pull request merged**
